closed/OpenJ9.gmk

```diff
@@ -517,7 +517,7 @@ run-preprocessors-j9 : stage-j9
 			OMR_DIR=$(OUTPUTDIR)/vm/omr \
 			SOURCETOOLS_DIR=$(call MixedPath,$(OPENJ9_TOPDIR))/sourcetools \
 			SPEC=$(OPENJ9_BUILDSPEC) \
-			UMA_OPTIONS_EXTRA="-buildDate $(shell date +'%Y%m%d')" \
+			UMA_OPTIONS_EXTRA="-buildDate $(shell $JAVA $TOPDIR/make/src/classes/DateUtil.java --format=yyyyMMdd)" \
```

Comment:
This must use `$(var)` to reference make macros; also suggest this can use `JAVA_SMALL`:
```
			UMA_OPTIONS_EXTRA="-buildDate $(shell $(JAVA_SMALL) $(TOPDIR)/make/src/classes/DateUtil.java --format=YYYYMMdd)" \
```

Reply:
Changed as suggested under: [b5249ac](https://github.com/ibmruntimes/openj9-openjdk-jdk/pull/684/commits/b5249aced00439e5fb5a923d0cea7118f1316d37)